### PR TITLE
fix according to user stories

### PR DESCRIPTION
**Pull request by mmanjuprasad**
I have fixed some pages according to user stories

### CODE REVIEW

**Comment by mmanjuprasad** [comment]
Changes made:
Changed the background image and added categories
Clear filters on product listing page
Increase or decrease button on products & price details
Wishlist add to cart button
Password toggle button on signup page
logout button on nav bar

File: cart&wishlist/cart.css [hunk 1]
```diff
@@ -108,4 +108,173 @@
...
+  }
+  
+  input#number {
+    text-align: center;
+    border: none;
+    border-top: 1px solid #ddd;
+    border-bottom: 1px solid #ddd;
+    margin: 0px;
+    width: 40px;
+    height: 40px;
```
Comment (Sahil25061999) [inline cart&wishlist/cart.css:164]: use rem

File: cart&wishlist/cart.css [hunk 2]
```diff
@@ -108,4 +108,173 @@
...
+  form #input-wrap {
+    margin: 0px;
+    padding: 0px;
+  }
+  
+  input#number {
+    text-align: center;
+    border: none;
+    border-top: 1px solid #ddd;
+    border-bottom: 1px solid #ddd;
```
Comment (Sahil25061999) [inline cart&wishlist/cart.css:161]: since the value is reporting you can use variables

File: cart&wishlist/cart.css [hunk 3]
```diff
@@ -108,4 +108,173 @@
...
+.delivery-det{
+    display: flex;
+    flex-direction: row;
+    justify-content:space-between;
+
+}
+.tot-amount-det{
+    display: flex;
+    flex-direction: row;
+    justify-content:space-between;
```
Comment (Sahil25061999) [inline cart&wishlist/cart.css]: this can be written together by using " , " separator.

File: cart&wishlist/cart.css [hunk 4]
```diff
@@ -108,4 +108,173 @@
...
+    width: 100%;
+    text-align: center;
+    margin-bottom:0.4rem;
+    margin-top:0.2rem;
+}
+.card-btn-checkout a{
+    border: none;
+    padding-left: 2rem;
+    padding-right: 2rem;
+    padding-top: 0.7em;
```
Comment (Sahil25061999) [inline cart&wishlist/cart.css]: i am assuming you have accidentally used em here, so you can fix this.

File: cart&wishlist/cart.css [hunk 5]
```diff
@@ -108,4 +108,173 @@
...
+    color: var(--cart-card-white-color);
+    background-color: #082032;
+    text-decoration: none;
+    cursor: pointer;
+    width: 100%;
+    font-size: 18px;
+    position: relative;
+    font-weight: 800;
+    top:1rem;
+    left:0rem;
```
Comment (Sahil25061999) [inline cart&wishlist/cart.css]: use just 0 no need to specify unit.

**Review by Sahil25061999 — APPROVED** [review]
A ui suggestion to either horizontally align the clear all button with the categories text or use it at the end of the all the filters , rather than using it inside the category section

File: styles.css [hunk 6]
```diff
@@ -141,12 +142,16 @@
     grid-template-columns: repeat(3, 1fr);
   }
 
-  #footer {
+  #footer{
+    postion: fixed;
     padding: 1rem;
     background-color: rgb(218, 218, 218);
     text-align: center;
     /* margin-top: 6rem; */
```
Comment (yaswanthmyneni) [inline styles.css:150]: changes: remove unused code

File: styles.css [hunk 7]
```diff
@@ -141,12 +142,16 @@
     grid-template-columns: repeat(3, 1fr);
   }
 
-  #footer {
+  #footer{
```
Comment (yaswanthmyneni) [inline styles.css:145]: changes: give styling to classes as much as possible.

File: styles.css [hunk 8]
```diff
@@ -197,7 +209,7 @@
   left: 40%;
   transform: translate(-50%, -50%);
   -ms-transform: translate(-50%, -50%);
-  background-color: rgb(66, 66, 66);
+  background-color: rgb(54, 53, 53);
   color: white;
   font-size: 16px;
```
Comment (yaswanthmyneni) [inline styles.css:214]: changes: use rem unit for values more than 15px

File: cart&wishlist/cart.css [hunk 9]
```diff
@@ -108,4 +108,173 @@
...
+    margin-top:0.2rem;
+}
+.card-btn-checkout a{
+    border: none;
+    padding-left: 2rem;
+    padding-right: 2rem;
+    padding-top: 0.7em; 
+    padding-bottom: 0.5rem;
+    color: var(--cart-card-white-color);
+    background-color: #082032;
```
Comment (yaswanthmyneni) [inline cart&wishlist/cart.css:237]: changes: use css variables for colors.

File: cart&wishlist/cart.css [hunk 10]
```diff
@@ -108,4 +108,173 @@
...
+    border-bottom:1px solid var(--cart-card-black-color);
+    padding-top:0.5rem;
+    padding-bottom:0.5rem;
+}
+
+.card-btn-checkout{
+    width: 100%;
+    text-align: center;
+    margin-bottom:0.4rem;
+    margin-top:0.2rem;
```
Comment (yaswanthmyneni) [inline cart&wishlist/cart.css]: changes: use px unit for values less than 1rem.

File: cart&wishlist/cart.css [hunk 11]
```diff
@@ -108,4 +108,173 @@
     height: 3rem;
     font-weight: 700;
     font-size: large;
-  }
\ No newline at end of file
+  }
+
+  form {
+    width: 200px;
```
Comment (yaswanthmyneni) [inline cart&wishlist/cart.css]: changes: use rem unit for values more than 15px. change all units.

**Review by yaswanthmyneni — APPROVED** [review]
You can merge after changes

**Pull request merged**